docs/wiki/[Examples]-Assign-a-Built-in-Policy.md

```diff
@@ -0,0 +1,191 @@
+## Overview
+
+This page describes how to assign built-in Azure Policies to your Enterprise Scale deployment.
+
+In this example you will use two built-in policies and one built-in policy set definition. You will use policies that are not already availble for assignment via the default  Enterprise Scale deployment.
+The policies you will use are `Not allowed resource types`  and `Deploy default Microsoft IaaSAntimalware extension for Windows Server`. The policy set definition (Initiative) you will use is  `NIST SP 800-53 Rev. 5`.
+
+>**NOTE**: You can view which policies are recommended to be assigned as part of an Enterprise Scale deployment [here](https://github.com/Azure/Enterprise-Scale/blob/main/docs/ESLZ-Policies.md).
+
+You will update the built-in configuration by following these steps:
+
+- Create the policy assignment files for `Not allowed resource types`, `Deploy default Microsoft IaaSAntimalware extension for Windows Server` and `NIST SP 800-53 Rev. 5`. You will also configure the `listOfResourceTypesNotAllowed` parameter within the `Not allowed resource types` assignment file to prevent the creation of Azure Firewalls
+- Assign the policy definition for `Deploy default Microsoft IaaSAntimalware extension for Windows Server` at the `es_root` Management Group by extending the built-in archetype for `es_root`
+- Assign the policy set definition for `NIST SP 800-53 Rev. 5` at the `es_root` Management Group by extending the built-in archetype for `es_root`
+- Assign the policy definition for `Not allowed resource types` at the `Landing Zones` Management Group by extending the built-in archetype for `es_landing_zones`.
+
+>IMPORTANT: To allow the declaration of custom or expanded templates, you must create a custom library folder within the root module and include the path to this folder using the `library_path` variable within the module configuration. In our example, the directory is `/lib`.
+
+In order to assign built-in policies, there needs to be an assignment file for each policy or policy set definition that we want to use.
+This module already includes assignment files for some built-in policies so it's important to check whether or not one exists before creating your own. You can do this by navigating to  `\modules\archetypes\lib\policy_assignments` and looking for an assignment file that matches the policy you want to assign.
```

Comment:
Provide a link to the lib https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/tree/main/modules/archetypes/lib

Reply:
Thanks Jack. I'll add this in the next commit.